src/controllers/instance-controller.js

```diff
@@ -9,14 +10,6 @@ const IVInstanceController = require('./instances/iv.js');
 const RedisClient = require('../services/redis.js');
 const Pokemon = require('../models/pokemon.js');
 
-const InstanceType = {
-    CirclePokemon: 'circle_pokemon',
-    CircleRaid: 'circle_raid',
-    CircleSmartRaid: 'circle_smart_raid',
-    AutoQuest: 'auto_quest',
-    PokemonIV: 'pokemon_iv'
-};
-
 class InstanceController {

```

Comment:
Seems that `/instances` returns more data than necessary?

<img width="644" alt="Screenshot 2020-09-06 at 20 40 43" src="https://user-images.githubusercontent.com/194908/92332946-946f7000-f081-11ea-81ba-a9f8d0f4a1a0.png">

Expected only the instances part

Reply:
That's from `/api/instances`?

Reply:
Yes, after been on page devices and assigments. Seems it caches something and includes everything in the last call

Reply:
Hmm looks like that's from the way I'm getting the locale information, the `src/data/default.js` file.

Reply:
Should be fixed now.

Reply:
Looks good